products/bigquery/api.yaml

```diff
@@ -1440,3 +1440,17 @@ objects:
                         - :INTEGER
                         - :FLOAT
                         - :BOOLEAN
+  - !ruby/object:Api::Resource
+    name: 'BigqueryTable'
```

Comment:
Hm, can you link the docs for this resource?  Under most circumstances, I think we would usually just call it Table, but I see that conflicts with the existing Table resource...

Reply:
Yes, we should have called it Table. But Table has been already used. Unfortunately Table has already been marked as exclude which means no resources including its children's will be generated. As its child resource, iam_policy can't be generated either. This is one way that we can use to bypass above limitation.

Reply:
I'd suggest that we should have a new option instead - "generate_iam_resources" or something like that - which is only considered if exclude == true, which would generate child resources anyway.  Does that make sense?  I'm not sure this workaround is the best way to do this.

Reply:
Agreed, adding an option to allow a child resource being created when its parent is set `exclude`, logically would be much clear.

Reply:
Hm, wait, isn't that what `exclude_resource` does?  Can you put `exclude_resource` on the existing Table resource and get the outcome you want?

Reply:
Yeah, in `terraform.yaml`, it seems like you might be able to remove `exclude: true` from Table and just move all the stuff you added to `BigqueryTable` (including `exclude_resource`), into `Table`.